web/auth/auth.go

```diff
@@ -45,6 +45,9 @@ func newUserSession(id uint64) *UserSession {
 	}
 }
 
+// TokenToUserMap
+var TokenToUserMap = make(map[string]uint64)
```

Comment:
Ideally, we want to avoid exporting the map directly. Instead, map accesses should be done via methods or functions controlled from this file.